cmd/codegen/hack/generate_and_sort_crds.sh

```diff
@@ -40,6 +40,7 @@ if ! $CONTROLLERGEN --version | grep -q "${CONTROLLERGEN_VERSION}" ; then
 fi
 
 # Run controller-gen
+${CONTROLLERGEN} object:headerFile=scripts/boilerplate.go.txt,year="$(date +%Y)" paths="./pkg/apis/..."
```

Comment:
This step creates `zz_generated` files. It's part of `kubebuilder`'s Makefile boilerplate.